### DISCUSSION TITLE
It would be possible to implement in the websites?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#646] [Q&A] [answered]
Is it possible to implement the Firefox UI Fix in the websites like Google, GitHub, Microsoft, or YouTube?

If it would be possible, then I'm asking if you are planning to implement the Firefox UI Fix into some websites?

**Comment by black7375** [comment, marked as answer]
There are too many things to manage.
Even now, it is almost limiting because I am splitting personal time to open source activities.

**Reply by Aleksandra0056** [reply]
But if someone wants to contribute, then you will accept pull requests from others? (I think pull requests related to GitHub, YouTube, Google, or other websites…)
